src/main/java/de/tum/in/www1/artemis/service/PrivacyStatementService.java

```diff
@@ -0,0 +1,129 @@
+package de.tum.in.www1.artemis.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.PrivacyStatement;
+import de.tum.in.www1.artemis.domain.PrivacyStatementLanguage;
+import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
+import de.tum.in.www1.artemis.web.rest.errors.InternalServerErrorException;
+
+@Service
+public class PrivacyStatementService {
+
+    private final Logger log = LoggerFactory.getLogger(PrivacyStatementService.class);
+
+    @Value("${artemis.legal-path}")
+    private String legalDirBasePath;
```

Comment:
```suggestion
    private Path legalDirBasePath;
```
Any reason for not using a proper path here? I think Spring configuration can handle the conversion internally.

Reply:
I wan't sure if Spring could handle the conversion to a path and in other places we used String, but Spring seems to be able to handle it, so I changed it.